py3status/modules/watson.py

```diff
@@ -0,0 +1,87 @@
+"""
+Display the current status of the watson time-tracking tool.
+
+Configuration Parameters:
+    cache_time: Number of seconds before state is re-read (default: 10)
+    watson_state_file: Path to the file which watson uses to track its own state (default: '~/.config/watson/state')
```

Comment:
same for default docstring

Reply:
we could also argue that this parameter is simply named `state_file` since we're in the watson module